src/QueryBuilder.php

```diff
@@ -38,6 +38,8 @@ final class QueryBuilder extends AbstractQueryBuilder
         SchemaInterface::TYPE_BINARY => 'BLOB',
         SchemaInterface::TYPE_BOOLEAN => 'NUMBER(1)',
         SchemaInterface::TYPE_MONEY => 'NUMBER(19,4)',
+        SchemaInterface::TYPE_UUID => 'RAW(16)',
+        SchemaInterface::TYPE_UUID_PK => 'RAW(16) DEFAULT SYS_GUID() PRIMARY KEY',
```

Comment:
Does `SYS_GUID()` give ordered keys?

Reply:
Yes. Crutch for using as default value. In Oracle default value should be set before "PRIMARY KEY".